Added rpc-http-max-request-content-length (#1363)
hyperledger/besu-docs: docs/public-networks/reference/cli/options.md

The maximum number of allowed HTTP JSON-RPC connections. Once this limit is reached, incoming connections are rejected. The default is 80.

### `rpc-http-max-request-content-length`

<!--tabs-->

# Syntax

```bash

> --rpc-http-max-request-content-length=<length>

Reviewer: ```suggestion
--rpc-http-max-request-content-length=<LONG>
```

Argument type in all caps, to match our other option descriptions.

```

> 

Reviewer: ````suggestion

# Example

```bash
--rpc-http-max-request-content-length=2097152
```

````

# Environment variable

```bash

> BESU_RPC_HTTP_MAX_REQUEST_CONTENT_LENGTH=<length>

Reviewer: ```suggestion
BESU_RPC_HTTP_MAX_REQUEST_CONTENT_LENGTH=2097152
```

```

# Configuration file

```bash

> rpc-http-max-request-content-length=<length>

Reviewer: ```suggestion
rpc-http-max-request-content-length=2097152
```

```

<!--/tabs-->

> The default body size limit is `5MB`. If you set the `rpc-http-max-request-content-length` option to a value greater than the default, the Besu node will only accept requests with a body size that is less than or equal to the specified value.

Reviewer: ```suggestion
The maximum request content length.
Besu only accepts JSON-RPC API requests with a body size less than or equal to this value.
The default is 5242880 (5 MB).
```

The maximum number of allowed HTTP JSON-RPC connections. Once this limit is reached, incoming connections are rejected. The default is 80.

### `rpc-http-max-request-content-length`

<!--tabs-->

# Syntax

```bash
--rpc-http-max-request-content-length=<LONG>
```

# Example

```bash
--rpc-http-max-request-content-length=2097152
```

# Environment variable

```bash
BESU_RPC_HTTP_MAX_REQUEST_CONTENT_LENGTH=2097152
```

# Configuration file

> ```bash

Reviewer: ```suggestion
```toml
```
